src/pubtools/_marketplacesvm/tasks/community_push/command.py

```diff
@@ -75,6 +75,7 @@ def raw_items(self) -> Iterator[AmiPushItem]:
                         )
                         continue
                     self.builds_borg.received_builds.add(item.build_info.id)
+                    item.dest = None if item.dest == "starmap" else item.dest
```

Comment:
Thanks!

Can you please do this for the `marketplace-push` as well? Then both pushes will take advantage of the staged structure.